source/AMDSClientUi.h

```diff
@@ -36,19 +36,17 @@ private slots:
 	void onNetworkSessionOpened();
 
 	/// slot to handle the signal of newServerConnected (add the serverIdentifier to the combox and update the ui displays -- buffernames and active connections)
-	void onNewServerConnected(QString serverIdentifier);
-	/// slot to handle the signal of serverError
-	void onServerError(int errorCode, QString serverIdentifier, QString errorMessage);
+	void onNewServerConnected(QString &serverIdentifier);
```

Comment:
Class-based parameters comment.  I believe these are covered in the last pull request, but just in case they aren't.